pandas/_libs/tslibs/conversion.pyx

```diff
@@ -594,15 +594,12 @@ cdef _TSObject convert_str_to_tsobject(str ts, tzinfo tz,
         obj.value = NPY_NAT
         obj.tzinfo = tz
         return obj
-    elif ts == "now":
+    elif ts in ("now", "today"):
```

Comment:
i think ATM we don't use a tuple here for perf reasons.  unless something has changed, cython constructs a tuple object

Reply:
I see. Changed back in the new commit.